lib/auth/join.go

```diff
@@ -295,7 +295,11 @@ func (a *Server) generateCerts(
 	}
 
 	// Emit audit event
-	log.Infof("Node %q [%v] has joined the cluster.", req.NodeName, req.HostID)
+	if req.Role == types.RoleInstance {
+		log.Infof("Node %q [%v] has joined the cluster. role=%s, systemRoles=%+v", req.NodeName, req.HostID, req.Role, systemRoles)
```

Comment:
Should we stop saying Node here?

```suggestion
		log.Infof("Instance %q [%v] has joined the cluster. role=%s, systemRoles=%+v", req.NodeName, req.HostID, req.Role, systemRoles)
```